runtests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script will execute all unit tests and integration tests under
```

Comment:
I love how you described the script, then later on said what the three environmental variables were. Looks really good, Jonathan. Solid A+ commenting 🤪

Reply:
Thanks!